Paco/src/com/pacoapp/paco/model/DatabaseHelper.java

```diff
@@ -419,5 +420,30 @@ private void updateJoinedExperiment(SQLiteDatabase db, Experiment experiment) {
 //                 + refCol + " = " + entry.getKey() + ";");
 //    }
 //  }
+  
+  public Cursor query(int tableIndicator, String[] projection, String selection,
+		  String[] selectionArgs, String sortOrder, String groupBy, String having) {
+	  SQLiteQueryBuilder qb = new SQLiteQueryBuilder();
+	  switch (tableIndicator) {
+	    case ExperimentProvider.EVENTS_OUTPUTS_DATATYPE:
+	    	qb.setTables(ExperimentProvider.EVENTS_TABLE_NAME+ " INNER JOIN " + "OUTPUTS" + 
```

Comment:
Please use the Experiment_Provider.OUTPUTS_TABLE_NAME constant for the table name here in case we change it in the future.